src/main/scala/nl/knaw/dans/easy/ps/DDM.scala

```diff
@@ -113,12 +113,35 @@ object DDM {
 
   def createComposedCreator(dictionary: Dictionary, authorFields: Iterable[(MdKey, String)]) = {
     <dcx-dai:creatorDetails>
-      <dcx-dai:author>
-        { authorFields.map(composedEntry(dictionary)) }
-      </dcx-dai:author>
+      {
```

Comment:
Mixed:
<dcx-dai:creatorDetails>
			<dcx-dai:organization>
				<dcx-dai:name xml:lang="zh-CN">DC.LPEeo</dcx-dai:name>
			</dcx-dai:organization>
		
			<dcx-dai:author>
		<dcx-dai:titles xml:lang="en-IE">dr.</dcx-dai:titles>
				<dcx-dai:initials>A</dcx-dai:initials>
				<dcx-dai:insertions>v.d</dcx-dai:insertions>
				<dcx-dai:surname>Coen</dcx-dai:surname>
				<dcx-dai:DAI>93313935x</dcx-dai:DAI> 
				<dcx-dai:organization>
					<dcx-dai:name xml:lang="zh-CN">A.C</dcx-dai:name>
				</dcx-dai:organization>
			</dcx-dai:author>
		</dcx-dai:creatorDetails>

Test:
it should "fail if person creators and organization-only creators are mixed" in {

    AddEmdToDatasetIngestDir("1", {enoughEmd +=
      ("DCX_CREATOR_ORGANIZATION" -> List("dans"))
    }) checkPreconditionsShould be(a[Success[_]])
  }